src/components/searchbar/Searchbar.vue

```diff
@@ -18,15 +20,60 @@
         {{ data.item.name }}
       </template>
       <template v-slot:append>
-        <v-icon>
-          {{ $route.path === "/detail" ? "mdi-magnify" : "mdi-help" }}
-        </v-icon>
+        <v-btn icon @click="search">
+          <v-icon>
+            {{ $route.path === "/detail" ? "mdi-magnify" : "mdi-help" }}
+          </v-icon>
+        </v-btn>
       </template>
     </v-autocomplete>
-  </v-toolbar>
+    <v-dialog
+        v-model="dialog"
+        persistent
+        max-width="600px"
+        class="rounded-xl"
+    >
+      <v-card class="pa-5 rounded-xl">
+        <v-card-title class="text-h5">
+          Anfrage
+        </v-card-title>
+        <v-form ref="form" v-model="valid" @submit.prevent="addMaterial">
+          <v-card-text>
+            <v-text-field v-model="material.name" :rules="nameRules" label="Name" required outlined/>
+            <v-textarea v-model="material.notes" label="Beschreibung" outlined/>
+            <v-alert v-if="message" color="error">
+              {{ message }}
+            </v-alert>
+          </v-card-text>
+          <v-card-actions class="px-4">
+            <v-spacer></v-spacer>
+            <v-btn
+                color="secondary"
+                class="rounded-xl"
+                text
+                @click="dialog = false"
+            >
+              Abbrechen
+            </v-btn>
+            <v-btn
+                :disabled="!valid"
+                :loading="$store.state.showLoadingSpinner"
+                color="primary"
+                type="submit"
+                class="rounded-xl"
```

Comment:
Hier bitte das rounded-xl auch raus nehmen weil unsere Chips sehen so aus, buttons haben weniger border-radius
```suggestion
                class="rounded-md"
```

Reply:
Aber dann müssen wir die in der Bottom Navigation auch ändern oder? 🙈

Reply:
Da wir überall sehr rund sind, finde ich die runden Buttons auch nicht schlecht.